Phonebook.Frontend/src/app/shared/components/user/user-list-card/user-list-card.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Person } from 'src/app/shared/models';
+
+@Component({
+  selector: 'app-user-list-card',
+  templateUrl: './user-list-card.component.html',
+  styleUrls: ['./user-list-card.component.scss']
+})
+export class UserListCardComponent {
+  @Input() public person: Person;
+  @Input() public actionButtonIcon: string;
+  @Input() public actionButtonClasses: string;
+  @Output() public actionButtonClicked: EventEmitter<any> = new EventEmitter<any>();
+
+  public buttonClicked() {
+    this.actionButtonClicked.emit();
+  }
+
+  public displayedUsername(): string {
```

Comment:
This is duplicate code

Reply:
Do you mean that what the function displayedUsername does is already in the html?

Reply:
No, the same function is used in other components (like other user-cards)

Reply:
So I should just rename it?